common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -257,131 +276,167 @@ div.video {
-            position: relative;
-            outline: $white dotted thin;
-            outline-offset: -2px;
-            overflow: auto;
-          }
-        }
+      .secondary-controls {
+        display: block;
+        float: left;
 
-        .menu-container {
-          float: left;
+        .speeds {
+          display: block;
           position: relative;
+          float: left;
+          border-right: 1px solid #666;
 
-          &.is-opened {
-            .menu {
-              display: block;
-              opacity: 1;
-              padding: 0;
-              margin: 0;
-              list-style: none;
+          .speed-button {
+            @extend %video-button;
+            @include clearfix();
+            border: none;
+            text-indent: 0;
+
+            @media (max-width: 1120px) {
+              min-width: 0;
+              width: 60px;
+            }
+
+            .icon {
+              float: left;
+              margin-top: 1em;
+              min-width: ($baseline / 2);
+            }
+
+            .label {
+              float: left;
+              padding: 0 lh(.25) 0 lh(.5);
+              font-size: em(14);
+              font-weight: normal;
+              letter-spacing: 1px;
+              text-transform: uppercase;
+              color: #999;
+
+              @media (max-width: 1120px) {
+                display: none;
+              }
+            }
+
+            .value {
+              float: left;
+              margin-bottom: 0;
+              padding: 0 lh(.5) 0 0;
+              font-weight: bold;
+              color: #fff;
+
+              @media (max-width: 1120px) {
+                padding: 0 lh(.5) 0 lh(.5);
+              }
             }
           }
 
-          .menu {
+          .video-speeds {
             @include transition(none);
-            @extend %ui-depth1;
-            box-shadow: inset 1px 0 0 #555, 0 1px 0 #444;
-            background-color: #444;
-            border: 1px solid $black;
-            bottom: 46px;
             display: none;
-            opacity: 0;
             position: absolute;
+            z-index: 10;
+            width: 127px;
+            bottom: 46px;
+            opacity: 0;
+            background-color: #444;
 
             li {
-              @extend %ui-fake-link;
-              box-shadow:  0 1px 0 #555;
-              border-bottom: 1px solid $black;
-              color: $white;
+              color: #fff;
+              cursor: pointer;
 
               a {
-                border: 0;
-                color: $white;
                 display: block;
-                padding: lh(0.5);
                 overflow: hidden;
+                padding: lh(.5);
                 text-overflow: ellipsis;
                 white-space: nowrap;
+                border: 0;
+                color: #fff;
 
-                &:hover, &:focus {
+                &:hover,
+                &:focus {
                   background-color: #666;
```

Comment:
I know you didn't touch these lines, but all the states should be the same colors, so please use $gray-d2 and $gray-l2 variables here (or something more accessible since those colors are pretty close).